.circleci/config.yml

```diff
@@ -129,6 +129,6 @@ commands:
           name: Create a release on GitHub
           command: |
             set -xe
-            pip3 install --user astronomer_e2e_test
-            # We can remove --debug after we are happy with it
-            astronomer-ci --debug publish-github-release --github-repository $CIRCLE_PROJECT_REPONAME --github-organization $CIRCLE_PROJECT_USERNAME --commitish $CIRCLE_SHA1
+            pyenv global 3.7.0
+            pip install astronomer_e2e_test
```

Comment:
pyenv will set pip to the correct python version. However, I found this did not work with --user flag